apps/src/studio/Item.js

```diff
@@ -60,7 +62,11 @@ Item.prototype.getDirectionFrame = function() {
     }
   }
 
-  return constants.frameDirTableWalkingWithIdle[this.displayDir];
+  var frameDirTable = this.spritesCounterclockwise ? 
+    constants.frameDirTableWalkingWithIdleCounterclockwise : 
+    constants.frameDirTableWalkingWithIdleClockwise;
```

Comment:
Cool.  Eventually, I'd really like to see this encapsulated into `StudioSpriteSheet` (or maybe a subclass of it that specializes in sheets where the animations correspond to directions) but we can do that refactor later.